index.js

```diff
@@ -1,88 +1,126 @@
 // ***** Question 1 *****
-// *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+let counter = 1
+counter += 1
+
```

Comment:
Nice :100: `let` allows us to reassign our variables

You can also just change `counter`'s value by doing `counter = 2`